AndroidNativeKotlinTemplate/app/src/main/AndroidManifest.xml

```diff
@@ -35,21 +36,23 @@
             'scheme' would be "sfdc", 'host' would be "login.salesforce.com",
             and 'path' would be "/oauth/done".
         -->
-        <!--
-        <activity android:name="com.salesforce.androidsdk.ui.LoginActivity"
-            android:theme="@style/SalesforceSDK"
-            android:launchMode="singleTop"
-            android:exported="true">
+        <!-- Use `android:name="com.salesforce.androidnativekotlintemplate.QrCodeEnabledLoginActivity"` when enabling log in via Salesforce UI Bridge API generated QR codes -->
+<!--        <activity-->
+<!--            android:name="com.salesforce.androidsdk.ui.LoginActivity"-->
```

Comment:
@brandonpage - For your previous comment on this section, this is functionally a match for `dev`, but I went with single-line commenting since it uncomments easier in Android Studio with Command-/ and also quotes the inner commented code regarding QR Code Log In.  How's that look?